xdsl/dialects/riscv_debug.py

```diff
@@ -0,0 +1,76 @@
+from collections.abc import Sequence, Set
```

Comment:
I think a short comment that this is a specific extension to `riscemu` would be reasonable here, just so that people don't mistake this for a standard riscv debugging extension.

Reply:
Ah nevermind, it's just below here^^